### DISCUSSION TITLE
CQ (WAZ) Map not up to date

### DISCUSSION THREAD

**Discussion by ve3hls** [magicbug/Cloudlog#2033] [General]
Looking at my CQ (Worked All Zones) map to see how many I need.  The map displayed shows none of Africa worked or verified.  In fact, I have verified every zone in Africa except for Zone 34, see the DXCC Africa map.

Also, the WAZ map shows I don't have Zone 40 (Greenland and Iceland) verified when in fact I have both countries verified on multiple bands.

Can you have a look and fix this please and thank you?

Ken, VE3HLS
![OX and TF](https://user-images.githubusercontent.com/81870838/225587328-869b6650-4ec7-4685-9887-607d216967ca.jpg)
![Africa QSLed](https://user-images.githubusercontent.com/81870838/225587329-d49ef77a-f76f-4ac4-8e1f-3fcffc445ef5.jpg)
![CQZoneMap](https://user-images.githubusercontent.com/81870838/225587330-4d2883e9-ef99-4ca8-8213-37354c88f9b9.jpg)

**Comment by AndreasK79** [comment]
As far as I know, there is nothing to fix in Cloudlog. You are the first to report such a problem with the map. I would start with checking the QSOs in your log, and see what CQ Zone is defined on each QSO.

**Comment by ve3hls** [comment]
100% of the QSOs I'm referring to are FT8 QSOs.  WSJT-X  does not exchange
zone information during the QSO, only grid squares, and I don't add the
zone data myself before uploading to Cloudlog, so I have to think that
Cloudlog is adding them after I upload them.  After that, it is ignoring
valid data and declaring that several verified zones have not been worked.

I don't have time to go through several thousand logs QSO-by-QSO to ensure
that zones have been correctly assigned.  As I said, I have multiple
verified QSOs on multiple bands for each zone that was incorrectly
identified on Cloudlog's zone map.  I sure can't tell what goes on behind
closed doors inside Cloudlog that would cause it to ignore valid QSO data
in the preparation of a report.

Sorry, but that's your job.

Ken  VE3HLS

**Comment by AndreasK79** [comment]
I'll make it my job when you pay me. The software is free. Everyone is free to do what they want. When you can't be bothered checking if your QSO has the CQ Zone set, why should I bother spending my free time looking into this?

**Comment by ve3hls** [comment]
When you can't be bothered to determine why your software is assigning
incorrect zone numbers to someone's logs then why bother sticking around?
I'll make it easy for you:  I'm gone.  I grabbed a copy of my logs and I'll
find some other software.  You're free to delete the logs and my account
anytime you like.  I won't be back.

Ken  VE3HLS
